services/greasyfork/greasyfork-downloads.service.js

```diff
@@ -0,0 +1,58 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseGreasyForkService from './greasyfork-base.js'
+
+class BaseGreasyForkInstalls extends BaseGreasyForkService {
+  static category = 'downloads'
+
+  static defaultBadgeData = { label: 'installs' }
+}
+
+class GreasyForkDailyInstalls extends BaseGreasyForkInstalls {
+  static route = { base: 'greasyfork/dd', pattern: ':scriptId' }
+
+  static examples = [
+    {
+      title: 'Greasy Fork',
+      namedParams: { scriptId: '407466' },
+      staticPreview: this.render({ downloads: 17 }),
+    },
+  ]
+
+  static _cacheLength = 21600
+
+  static render({ downloads }) {
+    return renderDownloadsBadge({ downloads, interval: 'day' })
+  }
+
+  async handle({ scriptId }) {
+    const data = await this.fetch({ scriptId })
+    return this.constructor.render({
+      downloads: data.daily_installs,
+    })
+  }
+}
+
+class GreasyForkTotalInstalls extends BaseGreasyForkInstalls {
+  static route = { base: 'greasyfork/dt', pattern: ':scriptId' }
+
+  static examples = [
+    {
+      title: 'Greasy Fork',
+      namedParams: { scriptId: '407466' },
+      staticPreview: this.render({ downloads: 3420 }),
+    },
+  ]
+
+  static _cacheLength = 21600
```

Comment:
Could you share your thinking on why you're overriding this?

The default value we use for download/install type badges is 900 which strikes a good balance between having fresh/accurate data and reducing spam against the upstream server. We typically don't increase it unless the latter becomes an issue

Reply:
It was copied from AMO as I used that as a starting point, I can remove it